### DISCUSSION TITLE
The OpenLAM Initiative

### DISCUSSION THREAD

**Discussion by njzjz** [deepmodeling/community#32] [General]
Peter Thiel once said, "We wanted flying cars, instead we got 140 characters (Twitter)." Over the past decade, we have made great strides at the bit level (internet), but progress at the atomic level (cutting-edge technology) has been relatively slow.

The accumulation of linguistic data has propelled the development of machine learning and ultimately led to the emergence of Large Language Models (LLMs). With the push from AI, progress at the atomic level is also accelerating. Methods like Deep Potential, by learning quantum mechanical data, have increased the space-time scale of microscopic simulations by several orders of magnitude and have made significant progress in fields like drug design, material design, and chemical engineering.

The accumulation of quantum mechanical data is gradually covering the entire periodic table, and the Deep Potential team has also begun the practice of the [DPA pre-training model](https://arxiv.org/abs/2208.08236). Analogous to the progress of LLMs, we are on the eve of the emergence of a general Large Atom Model (LAM). At the same time, we believe that open-source and openness will play an increasingly important role in the development of LAM.

Against this backdrop, the core developer team of Deep Potential is launching the OpenLAM Initiative to the community. This plan is still in the draft stage and is set to officially start on January 1, 2024. We warmly and openly welcome opinions and support from all parties.

The slogan for OpenLAM is "Conquer the Periodic Table!" We hope to provide a new infrastructure for microscale scientific research and drive the transformation of microscale industrial design in fields such as materials, energy, and biopharmaceuticals by establishing an open-source ecosystem around large microscale models. Relevant models, data, and workflows will be consolidated around the [AIS Square](https://www.aissquare.com/); related software development will take place in the DeepModeling open-source community. At the same time, we welcome open interaction from different communities in model development, data sharing, evaluation, and testing.

OpenLAM's goals for the next three years are: In 2024, to effectively cover the periodic table with first-principles data and achieve a universal property learning capability; in 2025, to combine large-scale experimental characterization data and literature data to achieve a universal cross-modal capability; and in 2026, to realize a target-oriented atomic scale universal generation and planning capability. Ultimately, within 5-10 years, we aim to achieve "Large Atom Embodied Intelligence" for atomic-scale intelligent scientific discovery and synthetic design.

OpenLAM's specific plans for 2024 include:
- Model Update and Evaluation Report Release:
  - Starting from January 1, 2024, driven by the Deep Potential team, with participation from all LAM developers welcomed.
  - Every three months, a major model version update will take place, with updates that may include model architecture, related data, training strategies, and evaluation test criteria.
- AIS Cup Competition:
  - Initiated by the Deep Potential team and supported by the [Bohrium Cloud Platform](https://bohrium.dp.tech/), starting in March 2024 and concluding at the end of the year;
  - The goal is to promote the creation of a benchmarking system focused on several application-oriented metrics.
- Domain Data Contribution:
  - Seeking collaboration with domain developers to establish "LAM-ready" datasets for pre-training and evaluation.
  - Domain datasets for iterative training of the latest models will be updated every three months.
- Domain Application and Evaluation Workflow Contribution:
  - The domain application and evaluation workflows will be updated and released every three months.
- Education and Training:
  - Planning a series of educational and training events aimed at LAM developers, domain developers, and users to encourage advancement in the field.

- How to Contact Us:
  - Direct discussions are encouraged in the DeepModeling community (here).
  - For more complex inquiries, please contact the project lead, Han Wang (王涵, [wang_han@iapcm.ac.cn](mailto:wang_han@iapcm.ac.cn), @wanghan-iapcm), Linfeng Zhang (张林峰, [zhanglf@aisi.ac.cn](mailto:zhanglf@aisi.ac.cn), @jameswind), for the new future of Science!

**Comment by link89** [comment]
Will this plan establish standards for data storage? The training data may come from different systems, software, and parameter settings, leading to various questions such as whether to retain raw output data or transformed data, the need to preserve input configurations, which configurations to retain, and so on.

NOMAD has its own data description system, but it feels overly complex and challenging to use. I think it would be helpful to establish a unified and user-friendly data storage specification.

**Reply by fandengdong** [reply]
Can't agree any more! A unified training data is the key to the OpenLAM plan.
